opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -32,25 +32,32 @@
 
 
 def _load_distros() -> BaseDistro:
-    for entry_point in iter_entry_points("opentelemetry_distro"):
-        try:
-            distro = entry_point.load()()
-            if not isinstance(distro, BaseDistro):
-                logger.debug(
-                    "%s is not an OpenTelemetry Distro. Skipping",
-                    entry_point.name,
-                )
-                continue
-            logger.debug(
-                "Distribution %s will be configured", entry_point.name
+    entry_points = iter_entry_points("opentelemetry_distro")
+    try:
+        first_distro = next(entry_points)
+
+        more_distros = [ep.module_name for ep in entry_points]
+
+        if more_distros:
+            logger.error(
+                "Multiple distros were found: (%s). Only one should be installed.",
+                ",".join([first_distro.module_name] + more_distros),
             )
-            return distro
-        except Exception as exc:  # pylint: disable=broad-except
-            logger.exception(
-                "Distribution %s configuration failed", entry_point.name
+            raise RuntimeError(
+                "Cannot Auto Instrument with multiple distros installed."
             )
-            raise exc
-    return DefaultDistro()
+
+        return first_distro.load()()
+    except StopIteration:
+        logger.warning(
+            "Initializing Auto Instrumentation without using a distro."
```

Comment:
A description of the error would be good.

Reply:
Added the following description:

```
logger.warning(
    "Failed to find even one installed distro. Initializing Auto Instrumentation without using a distro."
)
```